metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -137,4 +137,10 @@
         <value>yyyy.MM.dd.HH</value>
         <display-name>Elasticsearch Date Format</display-name>
     </property>
+    <property>
```

Comment:
Do you think It might be more appropriate to have this property in metron-rest-env instead?  Now when I change it I'm prompted to restart all Metron components when really the only one that needs to be restarted is REST and the UIs.  Both UIs depend on REST so they will also be included in the restart list.

Reply:
Yes, assuming this field is applicable only to the REST/UI/DAO, then this makes complete sense.  Will address this.